api/python/cell_census/tests/test_directory.py

```diff
@@ -60,6 +63,11 @@ def test_get_census_version_directory(directory_mock: Any) -> None:
         assert directory[tag] == cell_census.get_census_version_description(tag)
 
 
+def test_get_census_version_description_errors() -> None:
+    with pytest.raises(KeyError):
```

Comment:
Not strictly related to the tests, but I was wondering if maybe we don't want to specialize these exceptions? Right now everything is `KeyError` or `ValueError` but these exceptions map to very specific issues (for instance, non existent census version.

Reply:
sure, we could do that.

I agree it is a separate issue and I'll file it.